tests/utilities.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (division, absolute_import, unicode_literals,
+                        print_function)
```

Comment:
This file contains unused source code.

*PyUnusedCodeBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tests/utilities.py
+++ b/tests/utilities.py
@@ -3,7 +3,6 @@
 from __future__ import (division, absolute_import, unicode_literals,
                         print_function)
 
-import math
 import os
 import random
 import string
```